src/index.html

```diff
@@ -22,6 +22,394 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="header__logo"
```

Comment:
The value of the 'alt' attribute should describe the image. Consider using a more descriptive alternative text like 'Company Logo' instead of 'header__logo'.